### ISSUE TITLE
Settings for kopy.io

### ISSUE THREAD

**Issue by mjbrancato** [Komodo/KomodoEdit#1163]
### Short Summary
It would be nice to be able to set duration and privacy (Client-side encryption) in the Komodo IDE settings for kopy.io: https://disqus.com/home/discussion/komodoide/introducing_kopyio_komodo_ide_02/?utm_source=reply&utm_medium=email&utm_content=read_more#comment-2570921129

### Steps to Reproduce
N/A

### Expected results
N/A

### Actual results
N/A

### Platform Information
Komodo IDE 9.3
Xubuntu 15.10

*Komodo Edit or IDE?* 
*Komodo Version?* 
*Operating System (and version)?* 

### Additional Information

 -- eg. Error logs, screenshots, workarounds --

**Comment by mjbrancato** [comment]
I should probably create a new issue for this (minor), but it goes off of this (let me know if you want a new issue created for something like this).

Sometimes when I share with kopy,io, I get a http timeout or error, but if I try to share again it works. I don't use kopy,io all the time, but I will try and find the steps to reproduce. Nothing major, but just wanted to not this very minor issue.

**Comment by Naatan** [comment]
Yes, please create a new issue. I guess we could employ some type of retry mechanic.
